app/static/src/app/components/WodinPanels.vue

```diff
@@ -54,53 +48,80 @@ export default defineComponent({
 
         const modeClass = computed(() => modeClassMap[mode.value]);
 
-        const canCollapseLeft = computed(() => mode.value !== PanelsMode.Right);
-        const canCollapseRight = computed(() => mode.value !== PanelsMode.Left);
+        const panelWidth = ref<any>({});
+        const plotWidth = ref<any>({});
```

Comment:
Can we just call these 'leftWidth' and 'rightWidth'? They're both panels, and there might not be a plot in the right one! (Or could use 'options' and 'charts' at least!)

Can probably be a bit more specific than `any`..?

Reply:
done, ive gone with the options and charts width naming and created a type called WidthStyle